### PR TITLE
Add docs for Automation Credits

### PR DESCRIPTION
**Pull request by nettofarah**

### CODE REVIEW

File: automations/credits.mdx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+--- | --- | --- | ---
+| Free | 100 | 10 | 250 alerts /mo |
+| Starter | 100 | 10 | Unlimited |
+| Team | 100 | 10 | Unlimited |
+| Pro | Unlimited | Unlimited | Unlimited |
+| Business | Unlimited | Unlimited | Unlimited |
+
+## What happens when I run out of credits?
+
+When you run out of credits, your Automations will not run until you have more credits available.
```
Comment (fgrehm) [inline automations/credits.mdx:37]: Do we wanna talk about the specifics of aob? Like autopilot is paused, but eligibility is still xhecked

**Review by fgrehm — APPROVED** [review]

File: automations/credits.mdx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+There are separate pools for Actions, Auto Outbound and Slack Alerts, so you can try
+all three products individually on your own time.
+
+Credits are deducted when any succesfull Automation is run. 
+
+## Credit Limits
+
+- Slack credits are counted per month, and reset on the first day of your subscription start date.
+- Action and Auto Outbound credits are counted per workspace, and do not reset.
```
Comment (mshwery) [inline automations/credits.mdx:22]:
```suggestion
- Action and Auto Outbound credits are counted cumulatively for your workspace, and do not reset.
```

File: automations/credits.mdx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+limited credits.
+
+
+## Understanding Credits
+Each succesfull Automation run deducts 1 credit from your available credit pool.
+
+There are separate pools for Actions, Auto Outbound and Slack Alerts, so you can try
+all three products individually on your own time.
+
+Credits are deducted when any succesfull Automation is run.
```
Comment (mshwery) [inline automations/credits.mdx:17]: Is it when the run at all? Or when they deliver successfully?

Comment (mshwery) [reply automations/credits.mdx:17]: There's a difference in how these count for each product though:
- auto outbound = *autopilot* syncs the prospect to your destination sequence/campaign
- actions = successful delivery?
- slack alerts = successful delivery?

**Review by mshwery — APPROVED** [review]
nice! left some minor questions for clarification

File: automations/credits.mdx (hunk 1 above)
Comment (nettofarah) [reply automations/credits.mdx:37]: Good idea!

File: automations/credits.mdx [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+description: "Understand how Automation Credits are counted."
+---
+
+The Koala Automations product is available on all plans.
+The Business and Pro plans include unlimited action credits, while the other plans include
+limited credits.
+
+
+## Understanding Credits
+Each succesfull Automation run deducts 1 credit from your available credit pool.
```
[the remark this answers is not in the document]
Comment (nettofarah) [reply automations/credits.mdx:12]: Sounds good!

**Pull request merged**